eppo_metrics_sync/dbt_model_parser.py

```diff
@@ -131,7 +131,10 @@ def format(self):
         }
     
     def build(self):
-        self.parse_columns()
-        self.validate()
-        self.format()
-        return self.eppo_fact_source
+        if isinstance(self.model, dict):
```

Comment:
What if `self.model` isn't a dict? Should we explicitly handle that case? Currently it looks like this function would return `None` which could lead to cryptic error messages downstream if that's not something that should happen